### PR TITLE
Scale change

### PR DESCRIPTION
**Pull request by mattcushman**

### CODE REVIEW

**Comment by mattcushman** [comment]
Some minor improvements in gas and contract size
```new:
============ Deployment Bytecode Sizes ============
  ERC721Pool               -  23,976B  (97.55%)
  ERC20Pool                -  23,851B  (97.05%)
old:
============ Deployment Bytecode Sizes ============
  ERC721Pool               -  24,079B  (97.97%)
  ERC20Pool                -  23,954B  (97.47%)

new:
│ Function Name                              ┆ min             ┆ avg    ┆ median ┆ max     ┆ # calls │
│ addQuoteToken                              ┆ 99698           ┆ 162256 ┆ 141548 ┆ 666242  ┆ 48001   │
│ arbTake                                    ┆ 417835          ┆ 417925 ┆ 417925 ┆ 418015  ┆ 2       │
│ borrow                                     ┆ 165082          ┆ 224090 ┆ 188410 ┆ 795974  ┆ 104002  │
│ borrowerInfo                               ┆ 6895            ┆ 6895   ┆ 6895   ┆ 6895    ┆ 8001    │
│ inflatorInfo                               ┆ 2510            ┆ 2510   ┆ 2510   ┆ 2510    ┆ 8001    │
│ initialize                                 ┆ 120048          ┆ 120048 ┆ 120048 ┆ 120048  ┆ 12      │
│ interestRate                               ┆ 2341            ┆ 2341   ┆ 2341   ┆ 2341    ┆ 8001    │
│ kick                                       ┆ 163038          ┆ 237192 ┆ 232924 ┆ 1082354 ┆ 32000   │
│ loansInfo                                  ┆ 1816            ┆ 2685   ┆ 2684   ┆ 10684   ┆ 48015   │
│ moveQuoteToken                             ┆ 275025          ┆ 275025 ┆ 275025 ┆ 275025  ┆ 1       │
│ pledgeCollateral                           ┆ 61765           ┆ 62089  ┆ 62061  ┆ 536933  ┆ 96001   │
│ removeQuoteToken                           ┆ 155613          ┆ 174267 ┆ 174165 ┆ 207155  ┆ 4000    │
│ repay                                      ┆ 511349          ┆ 585671 ┆ 597294 ┆ 836873  ┆ 16001   │
│ take                                       ┆ 52648           ┆ 53635  ┆ 53364  ┆ 270486  ┆ 15998   │

old:
│ src/erc20/ERC20Pool.sol:ERC20Pool contract ┆                 ┆        ┆        ┆         ┆         │
│ Function Name                              ┆ min             ┆ avg    ┆ median ┆ max     ┆ # calls │
├╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌
│ addQuoteToken                              ┆ 99628           ┆ 162404 ┆ 141478 ┆ 738006  ┆ 48001   │
│ arbTake                                    ┆ 426443          ┆ 426533 ┆ 426533 ┆ 426623  ┆ 2       │
│ borrow                                     ┆ 166228          ┆ 228026 ┆ 191545 ┆ 799137  ┆ 104002  │
│ borrowerInfo                               ┆ 6895            ┆ 6895   ┆ 6895   ┆ 6895    ┆ 8001    │
│ inflatorInfo                               ┆ 2510            ┆ 2510   ┆ 2510   ┆ 2510    ┆ 8001    │
│ initialize                                 ┆ 120048          ┆ 120048 ┆ 120048 ┆ 120048  ┆ 12      │
│ interestRate                               ┆ 2341            ┆ 2341   ┆ 2341   ┆ 2341    ┆ 8001    │
│ kick                                       ┆ 171327          ┆ 249924 ┆ 244623 ┆ 1091298 ┆ 32000   │
│ loansInfo                                  ┆ 1816            ┆ 2685   ┆ 2684   ┆ 10684   ┆ 48015   │
│ moveQuoteToken                             ┆ 321865          ┆ 321865 ┆ 321865 ┆ 321865  ┆ 1       │
│ pledgeCollateral                           ┆ 62338           ┆ 64765  ┆ 64559  ┆ 539523  ┆ 96001   │
│ removeQuoteToken                           ┆ 147555          ┆ 176717 ┆ 176621 ┆ 203248  ┆ 4000    │
│ repay                                      ┆ 513939          ┆ 588547 ┆ 600457 ┆ 840036  ┆ 16001   │
│ take                                       ┆ 53221           ┆ 56857  ┆ 56237  ┆ 277428  ┆ 15998   │
```

File: src/libraries/Deposits.sol [hunk 1]
```diff
@@ -109,7 +108,7 @@ library Deposits {
             // Update node effected by addition.
             } else {
                 scaled = self.scaling[index];
-                if (scaled != 0) sc = Maths.wmul(sc, scaled);
+                if (scaled!=0) sc = Maths.wmul(sc, scaled);
```
Comment (ith-harvey) [inline src/libraries/Deposits.sol]: NIT: add spacing back : )

File: src/libraries/Deposits.sol [hunk 2]
```diff
@@ -131,19 +130,16 @@ library Deposits {
...
         uint256 sc    = Maths.WAD;
-        uint256 index = 4096;
-
-        uint256 scaledM;
+        uint256 index = m_ + i;
         uint256 scaledMInc;
-        uint256 ssCond;
-
+        uint256 sValue;
+
```
Comment (ith-harvey) [inline src/libraries/Deposits.sol]: NIT: get rid of white space

File: src/libraries/Deposits.sol [hunk 3]
```diff
@@ -131,19 +130,16 @@ library Deposits {
...
         uint256 scaledMInc;
-        uint256 ssCond;
-
+        uint256 sValue;
+        
         while (i > 0) {
             scaledMInc = self.scaling[index];
-            ssCond = scaledMInc != 0 ? ss + Maths.wmul(Maths.wmul(sc, scaledMInc), self.values[index]) : ss + Maths.wmul(sc, self.values[index]);
-            if (ssCond < x_) {
+            sValue = (scaledMInc!=0)?ss + Maths.wmul(Maths.wmul(sc, scaledMInc), self.values[index]) : ss + Maths.wmul(sc, self.values[index]);
```
Comment (ith-harvey) [inline src/libraries/Deposits.sol]: NIT: spacing inside of parenthesis and turnary

File: src/libraries/Deposits.sol [hunk 4]
```diff
@@ -187,16 +183,19 @@ library Deposits {
...
 
         while (i_ > 0) {
             scaledI =  self.scaling[i_];
             
             // Calc sum, will only be stored in range parents of starting node, i_
-            sum = scaledI != 0 ? sum + Maths.wmul(Maths.wmul(df, self.values[i_]), scaledI) : sum + Maths.wmul(df, self.values[i_]);
-
-            // Apply scaling to all range parents less then starting node, i_
-            self.scaling[i_] = scaledI != 0 ? Maths.wmul(f_, scaledI) : f_;
+            if(scaledI!=0) {
```
Comment (ith-harvey) [inline src/libraries/Deposits.sol]: NIT: spacing

File: src/libraries/Deposits.sol [hunk 5]
```diff
@@ -210,8 +209,8 @@ library Deposits {
 
                 // Sum > 0 only when j is a range parent of starting node, i_.
                 self.values[j] += sum;
-                scaledJ = self.scaling[j];
-                if (scaledJ != 0) sum = Maths.wmul(sum, scaledJ);
+                scaledI = self.scaling[j];
+                if (scaledI!=0) sum = Maths.wmul(sum, scaledI);
```
Comment (ith-harvey) [inline src/libraries/Deposits.sol]: NIT: spacing

File: src/libraries/Deposits.sol [hunk 6]
```diff
@@ -288,7 +282,7 @@ library Deposits {
             // Update node effected by removal.
             } else {
                 scaled = self.scaling[index];
-                if (scaled != 0) sc = Maths.wmul(sc, scaled);
+                if (scaled!=0) sc = Maths.wmul(sc,scaled);
```
Comment (ith-harvey) [inline src/libraries/Deposits.sol]: NIT: Spacing

File: src/libraries/Deposits.sol [hunk 7]
```diff
@@ -326,20 +318,20 @@ library Deposits {
...
+        uint256 scaled;
 
         i_         += 1;
         s_         =  self.values[i_];
 
-        uint256 scaled;
         while (j & k != 0) {
             scaled = self.scaling[j];
-            s_ = scaled != 0 ? s_ - Maths.wmul(scaled, self.values[j]) : s_ - self.values[j];
+            s_ = scaled!=0 ? s_ - Maths.wmul(scaled, self.values[j]) : s_ - self.values[j];
```
Comment (ith-harvey) [inline src/libraries/Deposits.sol]: NIT: spacing

File: src/libraries/Deposits.sol [hunk 8]
```diff
@@ -326,20 +318,20 @@ library Deposits {
...
             j  = j - k;
             k  = k << 1;
         }
         while (i_ <= SIZE) {
             scaled = self.scaling[i_];
-            if (scaled != 0) s_ = Maths.wmul(scaled, s_);
+            if(scaled!=0) s_ = Maths.wmul(scaled, s_);
```
Comment (ith-harvey) [inline src/libraries/Deposits.sol]: NIT: spacing

**Review by ith-harvey — COMMENTED** [review]
Just dropped some nit picky comments... suggest we stick to the spacing formatting we have everywhere else in the contracts : )

**Review by grandizzy — APPROVED** [review]
wow, not so ` minor improvements` :) looking great! LGTM, just fix Ian's comments and good to merge

**Review by EdNoepel — APPROVED** [review]
LGTM

**Pull request merged**